<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert_s4</class>
 <widget class="QDialog" name="insert_s4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemNo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>UserNo</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_userNo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_itemNo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_quantity">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_itemNo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemNo Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_confirm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_zipdoe">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ZipCode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_userNo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>UserNo Check</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
